generators/entity-client/templates/angular/src/main/webapp/app/entities/_entity.service.ts

```diff
@@ -67,16 +65,12 @@ export class <%= entityAngularName %>Service {
     <%_ } _%>
         const copy = this.convert(<%= entityInstance %>);
         return this.http.put<<%= entityAngularName %>>(this.resourceUrl, copy, { observe: 'response' })
-            .map((res: HttpResponse<<%= entityAngularName %>>) => {
-                return this.convertResponse(res);
```

Comment:
i broke the line and indented this because linter complains about exceeding line length with the long names of the pagination-with-infinite-scroll  entities in tests

Reply:
oh ok, I guess its fine to break line as below as well
```
.map((res: HttpResponse<<%= entityAngularName %>>) =>
    this.convertResponse(res));
```

Reply:
Another way to reduce length is to assign a type for this and use that so
```
export type EntityResponseType = HttpResponse<<%= entityAngularName %>>;
...
.map((res: EntityResponseType) => this.convertResponse(res));
```